ptx/S_12_4_FTCLI.ptx

```diff
@@ -8,144 +8,104 @@
       <ul>
         <li>
           <p>
-            How can we characterize the vector fields <m>\vF</m> for which
-            <m>\int_C\vF\cdot d\vr</m> has the same value for every oriented path
-            from a point <m>P</m> to a point <m>Q</m>?
+          Characterize the vector fields <m>\vF</m> for which <m>\int_C\vF\cdot d\vr</m> has the same value for every oriented path from a point <m>P</m> to a point <m>Q</m>?
```

Comment:
Is there a reason that these were rewritten to be statements rather than questions? In the rest of Active Calculus (and the other three sections), the `objectives` element is `rename`d to be "Motivating Questions", so I think that's what we should be going for here.